tests/std/tests/P2286R8_text_formatting_tuple/test.cpp

```diff
@@ -435,10 +435,32 @@ auto test_vformat_exception = []<class CharT, class... Args>([[maybe_unused]] st
     }
 };
 
+// Also test that functions taking non-constructible basic_format_context specializations can be well-formed,
+// despite that they can't be actually called.

```

Comment:
Grammar nitpicks: "despite that" doesn't sound right; I would expect "despite the fact that". (https://en.wiktionary.org/wiki/despite notes this.) Also, "can't be actually called" should be "can't actually be called". English is super weird :joy_cat: